### PR TITLE
net-analyzer/python-gvm: new package

### PR DESCRIPTION
**Pull request by hsntgm**
Package-Manager: Portage-2.3.79, Repoman-2.3.16
Signed-off-by: Hasan ÇALIŞIR <hasan.calisir@psauxit.com>

### CODE REVIEW

File: net-analyzer/python-gvm/python-gvm-1.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
```
Comment (juippis) [inline net-analyzer/python-gvm/python-gvm-1.1.0.ebuild]: Please no python3.5 for new packages, it's getting removed soon.

File: net-analyzer/python-gvm/python-gvm-1.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/paramiko[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+DEPEND="
+	${RDEPEND}"
+
+src_prepare() {
+	distutils-r1_python_prepare_all
+	# Exlude tests & docs
```
Comment (juippis) [inline net-analyzer/python-gvm/python-gvm-1.1.0.ebuild]: But why? We'd prefer having them both enabled.

File: net-analyzer/python-gvm/python-gvm-1.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+SLOT="0"
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-python/defusedxml[${PYTHON_USEDEP}]
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/paramiko[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (juippis) [inline net-analyzer/python-gvm/python-gvm-1.1.0.ebuild]: `setuptools` is usually in BDEPEND, it **can** be in RDEPEND (please verify it belongs there) but most definitely not in DEPEND.

File: net-analyzer/python-gvm/python-gvm-1.1.0.ebuild (hunk 1 above)
Comment (hsntgm) [reply net-analyzer/python-gvm/python-gvm-1.1.0.ebuild]: done

File: net-analyzer/python-gvm/python-gvm-1.1.0.ebuild (hunk 2 above)
Comment (hsntgm) [reply net-analyzer/python-gvm/python-gvm-1.1.0.ebuild]: because of build issue

File: net-analyzer/python-gvm/python-gvm-1.1.0.ebuild (hunk 3 above)
Comment (hsntgm) [reply net-analyzer/python-gvm/python-gvm-1.1.0.ebuild]: now in BDEPEND

**Comment by hsntgm** [comment]
ping

File: net-analyzer/python-gvm/python-gvm-1.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+inherit distutils-r1
+
+DESCRIPTION="Greenbone Vulnerability Management Python Library"
+HOMEPAGE="https://www.greenbone.net/en/"
```
Comment (juippis) [inline net-analyzer/python-gvm/python-gvm-1.1.0.ebuild]: Also please add the Github page as 2nd homepage when you next time bump this package.

**Pull request closed without merging**